addons/common/fnc_isAttenuated.sqf

```diff
@@ -0,0 +1,78 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_isAttenuated
+
+Description:
+    Checks whether a unit is attenuated.
+
+Parameters:
+    _unit - Unit to check <OBJECT>
+
+Returns:
+    "true" for inside unit that cant hear the outside <BOOL>
+
+Examples:
+    (begin example)
+        if ([player] call CBA_fnc_isAttenuated) then {
```

Comment:
Conflicts with L12.
Line 12 says returns STRING "true" (which is wrong datatype btw) for when unit cant hear the outside.
This example says it returns true when the unit can hear the ourside. What now?

Reply:
my fail... need someone for the header infos... my english is trash

Reply:
I'd just do:
`_isAttenuated = [player] call CBA_fnc_isAttenuated;`
Seems more in form with other headers.